activesupport/lib/active_support/duration.rb

```diff
@@ -86,7 +93,7 @@ def eql?(other)
       Duration === other && other.value.eql?(value)
     end
 
-    def hash
+    def hash # :nodoc:
```

Comment:
To me this should be left public, so people know a duration can be embedded in a collection. But don't bother documenting it. Let's rely on users knowing Ruby semantics instead.